config/config.go

```diff
@@ -1235,11 +1237,11 @@ func DefaultInstrumentationConfig() *InstrumentationConfig {
 		PrometheusListenAddr: ":26660",
 		MaxOpenConnections:   3,
 		Namespace:            "cometbft",
-		InfluxURL:            "",
-		InfluxOrg:            "celestia",
-		InfluxBucket:         "e2e",
-		InfluxBatchSize:      20,
-		InfluxTables:         DefaultInfluxTables,
+		TracePushURL:         "",
+		TraceOrg:             "celestia",
+		TraceDB:              "e2e",
+		TraceBufferSize:      20,
```

Comment:
I feel like this should be a lot larger than 20. Maybe 100 and then call flush after a proposal is sent (or some other period where we are usually waiting and not blocking the network)

Reply:
good catch, put it at 1k, but we might even want to go higher depending on what we're tracing or if we see that we block ever